HIP/hip-412.md

```diff
@@ -104,19 +110,27 @@ https://w3c.github.io/did-core/
 
 #### image
 
-Points to the uri of the image file. See **[uri formatting]** section for more details.
+A URI pointing to a resource with mime type image/* representing the asset to which this token represents. Consider making any images at a width between 320 and 1080 pixels and aspect ratio between 1.91:1 and 4:5 inclusive. See **[uri formatting]** section for more details.
+
+The `image` field is required. It can both serve as a preview image or the default image for your NFT to ensure cross-platform compatibility. The image will be displayed in wallets and marketplaces by default. Some platforms may support displaying other file types such as 3D files, audio or video. Use the `is_default_file` in your `files` array to mark a file as the one you would prefer to have displayed by default if the NFT tooling supports it. In this case, the `image` field serves as a preview image or 'thumbnail'.
```

Comment:
For single-image NFTs, creators are encouraged to point to a thumbnail in the `image` field, and put the full resolution image in the `files` array with the `is_default_file` boolean set.

Reply:
Updated the description with this recommendation 👍